tests/device_event/device_event_api.cpp

```diff
@@ -34,42 +38,63 @@ class TEST_NAME : public util::test_base {
       {
         auto testQueue = util::get_cts_object::queue();
 
-        int data = 1234;
+        constexpr size_t bufferSize = 512;
+        constexpr size_t sensorIndex = bufferSize / 2;
```

Comment:
Why `sensorIndex`? And later you talk about "sensor data".

Reply:
Because we are verifying a single item in a middle of the buffer. It works like a failure sensor somehow. Should I change a naming?

Reply:
I don't really mind, I was just curious because I didn't really associate this with sensors.